src/js/pre-need/config/form.js

```diff
@@ -454,21 +455,25 @@ const formConfig = {
             application: {
               claimant: {
                 desiredCemetery: autosuggest.uiSchema(
-                  'Which VA National Cemetery would you prefer to be buried in?',
+                  'Which VA national cemetery would you prefer to be buried in?',
                   getCemeteries
                 ),
                 'view:desiredCemeteryNote': {
-                  'ui:description': (
-                    <div className="usa-alert usa-alert-info no-background-image">
-                      <strong>Please note:</strong> This doesn't guarantee you'll be buried in your preferred cemetery. We'll try to fulfill your wishes, but will assign a gravesite in a cemetery with available space at the time of need.
-                    </div>
-                  )
+                  'ui:description': desiredCemeteryNote
                 }
               },
               hasCurrentlyBuried: {
-                'ui:title': 'Is there anyone currently buried in a VA National Cemetery under your eligibility?',
+                'ui:title': 'Is there anyone currently buried in a VA national cemetery under your eligibility?',
                 'ui:widget': 'radio',
                 'ui:options': {
+                  updateSchema: (formData, schema, uiSchema) => {
+                    if (!isVeteran(formData)) {
+                      /* eslint-disable no-param-reassign */
+                      uiSchema['ui:title'] = 'Is there anyone currently buried in a VA national cemetery under your sponsor’s eligibility?';
```

Comment:
I think we should try to avoid modifying `uiSchema`. You can set a `title` property on the schema to set the label (you'll have to remove the `ui:title` property in `uiSchema` for that to work).

Reply:
Thanks for the tip @jbalboni! Updated to conditionally assign a title to schema based on `isVeteran`.